SpatialGDK/Source/SpatialGDK/Public/Schema/Interest.h

```diff
@@ -54,6 +54,7 @@ struct QueryConstraint
 	TSchemaOption<uint32> ComponentConstraint;
 	TArray<QueryConstraint> AndConstraint;
 	TArray<QueryConstraint> OrConstraint;
+	// TSchemaOption<> SelfConstraint; TODO
```

Comment:
Is there a follow up ticket for this?

Reply:
https://github.com/spatialos/UnrealGDK/pull/2686